### PR TITLE
more strings

### PR DESCRIPTION
**Pull request by LWinterberg**
* Resolves: #5092
* Renames several other features: 
    * Overdub -> Hear other tracks during recording
    * Time-Shift -> move clip
    * Sync-Lock -> keep tracks synchronized
    * Software playthrough -> input monitoring
    * Keyboard (on preferences) -> Shortcuts
* Adds undo messages to trimming

### CODE REVIEW

File: src/menus/ExtraMenus.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ BaseItemSharedPtr ExtraMiscItems()
 
          return (
          // Accel key is not bindable.
-         Command( wxT("FullScreenOnOff"), XXO("&Full Screen (on/off)"),
+         Command( wxT("FullScreenOnOff"), XXO("Enable &Full Screen"),
```
Comment (Paul-Licameli) [inline src/menus/ExtraMenus.cpp:72]: Ditto?

File: src/menus/TrackMenus.cpp [hunk 2]
```diff
@@ -1273,7 +1273,7 @@ BaseItemSharedPtr TracksMenu()
       ),
 
       Section( "",
-         Command( wxT("SyncLock"), XXO("Sync-&Lock Tracks (on/off)"),
+         Command( wxT("SyncLock"), XXO("Keep tracks synchronized (Sync &Lock)"),
```
Comment (Paul-Licameli) [inline src/menus/TrackMenus.cpp:1276]: Again

File: src/menus/TransportMenus.cpp [hunk 3]
```diff
@@ -827,14 +827,14 @@ BaseItemSharedPtr TransportMenu()
                   XXO("Sound Activation Le&vel..."), OnSoundActivated,
                   AudioIONotBusyFlag() | CanStopAudioStreamFlag() ),
                Command( wxT("SoundActivation"),
-                  XXO("Sound A&ctivated Recording (on/off)"),
+                  XXO("Enable Sound A&ctivated Recording"),
```
Comment (Paul-Licameli) [inline src/menus/TransportMenus.cpp:830]: Noch einmal

File: src/AdornedRulerPanel.cpp [hunk 4]
```diff
@@ -2919,7 +2919,7 @@ using namespace MenuTable;
 using Options = CommandManager::Options;
 AttachedItem sAttachment{
    { wxT("Transport/Other/Options/Part2"), { OrderingHint::Begin, {} } },
-   Command( wxT("PinnedHead"), XXO("Pinned Play/Record &Head (on/off)"),
+   Command( wxT("PinnedHead"), XXO("Enable Pinned Play &Head"),
```
[the remark this answers is not in the document]
Comment (LWinterberg) [reply src/AdornedRulerPanel.cpp:2922]: a) I'm not sure about most of the words in this PR, but wanted to get a build for @Tantacrul @DilsonsPickles et al. to see it in context
b) "Enable..." items are usually quite clear: When you click the menu entry solo, clicking it enables it. When you revisit it, it's got a checkmark next to it, indicating it's enabledness, and clicking it again disables it. This is mostly consistent with how checkboxes in the preferences work,  it's just that the empty box is missing.

[the remark this answers is not in the document]
Comment (Paul-Licameli) [reply src/AdornedRulerPanel.cpp:2922]: As you like

**Comment by LWinterberg** [comment]
strings got reviewed earlier, though it seems like I screwed up the rebase onto master. I wonder what happened there.

File: src/menus/TransportMenus.cpp [hunk 5]
```diff
@@ -827,14 +827,14 @@ BaseItemSharedPtr TransportMenu()
...
                   OnToggleSoundActivated,
                   AudioIONotBusyFlag() | CanStopAudioStreamFlag(),
                   Options{}.CheckTest(SoundActivatedRecord) )
             ),
 
             Section( "Part2",
-               Command( wxT("Overdub"), XXO("&Overdub (on/off)"),
+               Command( wxT("Overdub"), XXO("Hear other tracks during recording"),
```
Comment (Paul-Licameli) [inline src/menus/TransportMenus.cpp:837]: No shortcut key now

File: src/prefs/RecordingPrefs.cpp [hunk 6]
```diff
@@ -103,7 +103,7 @@ void RecordingPrefs::PopulateOrExchange(ShuttleGui & S)
                     {wxT("/AudioIO/Playthrough"),
                      false});
 #endif
-      S.TieCheckBox(XXO("&Software playthrough of input"),
+      S.TieCheckBox(XXO("Audible input &monitoring"),
```
Comment (Paul-Licameli) [inline src/prefs/RecordingPrefs.cpp:106]: Another changed shortcut, conflict?

File: src/menus/TransportMenus.cpp (hunk 5 above)
Comment (LWinterberg) [reply src/menus/TransportMenus.cpp]: re-added it

File: src/menus/TransportMenus.cpp [hunk 7]
```diff
@@ -844,7 +844,7 @@ BaseItemSharedPtr TransportMenu()
                      true
 #endif
                   ) ),
-               Command( wxT("SWPlaythrough"), XXO("So&ftware Playthrough (on/off)"),
+               Command( wxT("SWPlaythrough"), XXO("Enable audible input &monitoring"),
```
[the remark this answers is not in the document]
Comment (LWinterberg) [reply src/menus/TransportMenus.cpp:847]: yes, no conflict

File: src/prefs/RecordingPrefs.cpp (hunk 6 above)
Comment (LWinterberg) [reply src/prefs/RecordingPrefs.cpp:106]: no conflict

**Review by Paul-Licameli — APPROVED** [review]

**Comment by dozzzzer** [comment]
Tested

**Comment by Paul-Licameli** [comment]
> Tested

Including tests of the accelerator keys?

**Comment by dozzzzer** [comment]
> Including tests of the accelerator keys?

Yes

**Pull request merged**